### DISCUSSION TITLE
codespaces should have something like this

### DISCUSSION THREAD

**Discussion by manuthecoder** [community/community#41999] [Codespaces] [answered]
### Select Topic Area

Product Feedback

### Body

GitPod sends a notification when a port is available. This would be extremely beneficial if Codespaces had this as well. 

![image](https://user-images.githubusercontent.com/77016441/208256307-5be0def3-e9b4-4df7-b243-0d95953ce64f.png)

**Comment by cmbrose** [comment, marked as answer]
Codespaces actually does already have this toast when a port is forwarded:

![image](https://user-images.githubusercontent.com/5447118/208712378-7e39cad0-aa8e-422c-9ef0-07b1c16dc6b0.png)

There are a few vscode settings that may cause the toast to not appear - for example, disabling auto-forwarding, or configuring a port to be ignored when forwarded may cause the toast to not appear.

![image](https://user-images.githubusercontent.com/5447118/208713371-1c3c5987-712a-4bd3-8b6b-18e44661f8bb.png)
